package.json

```diff
@@ -28,20 +28,24 @@
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
     "babel-eslint": "^10.1.0",
+    "bootstrap": "^5.0.0-beta3",
     "broccoli-asset-rev": "^3.0.0",
+    "ember-ajax": "^5.0.0",
```

Comment:
You need to be careful when adding addons and in this case this is 2-3 years old and that's not maintained anymore.

Reply:
Yes i will remove it